prow/cmd/sub/OWNERS

```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- chaodaiG
+- cjwagner
+- listx
+- sebastienvas
```

Comment:
did we check with seb? 😅

Reply:
synced offline with Seb, he doesn't have the bandwidth, removed